ui/apps/platform/src/Components/CompoundSearchFilter/components/CompoundSearchFilter.tsx

```diff
@@ -87,9 +97,33 @@ function CompoundSearchFilter({
                     setInputValue(value);
                 }}
                 onSearch={(value) => {
-                    // @TODO: Add search filter value to URL search
-                    // eslint-disable-next-line no-alert
-                    alert(value);
+                    if (selectedEntity && selectedAttribute) {
+                        const entityObject = config[selectedEntity];
+                        const attributeObject: SearchFilterAttribute =
+                            entityObject?.attributes[selectedAttribute];
+                        const { inputType } = attributeObject;
+
+                        let result = '';
+
+                        if (inputType === 'text') {
+                            result = ensureString(value);
+                        } else if (inputType === 'condition-number') {
+                            const { condition, number } = ensureConditionNumber(value);
+                            result = `${conditionMap[condition]}${number}`;
+                        } else if (inputType === 'autocomplete') {
+                            result = ensureString(value);
+                        } else if (inputType === 'date-picker') {
+                            result = ensureString(value);
+                        } else if (inputType === 'select') {
+                            const selection = ensureStringArray(value);
+                            result = selection.join(',');
```

Comment:
This might cause problems when setting the value via URL. I believe that API will expect either a string, or a `string[]`, and I'm not sure what the semantics will be if we pass it an array of values concatenated with `,` characters.

I can test this out in a little bit, but it might be better to have `onSearch` take a `string[]` as a second option, and just pass an array with a single value for all of the other input types above.

Reply:
Yep - here is an example of what is happening when adding values to the current `searchFilter`:
![image](https://github.com/stackrox/stackrox/assets/1292638/00c75718-fb7d-4d0d-b5db-947c30fa429e)

Reply:
Ah, I see. Then we can pass it as an array instead 👍🏼